src/options.cpp

```diff
@@ -421,3 +408,24 @@ void OptionParser::parseEmitPragmas(std::string val) {
     else
         printHelpAndExit("Can't recognize emit-pragmas use level");
 }
+
+void OptionParser::parseOutDir(std::string val) {
+    Options &options = Options::getInstance();
+    options.setOutDir(std::move(val));
+}
+
+void Options::dump(std::ostream &stream) {
+    dumpVersion(stream);
+    stream << "Seed: " << seed << "\n";
+    stream << "Invocation: ";
+    for (const auto &option : raw_options) {
+        stream << option << " ";
```

Comment:
Avoid trailing white space - print it before the argument, not after.

Reply:
Done